### ISSUE TITLE
Rule proposal: jsx-padded-children

### ISSUE THREAD

**Issue by DimitarNestorov** [yannickcr/eslint-plugin-react#2045]
Something similar to [`padded-blocks`](https://eslint.org/docs/rules/padded-blocks) where it disallows code similar to this:
```html
<div>

  <img />
</div>
```

Depending on the setting the following should **not** throw a warning:
```html
// jsx-padded-children: [2, "always"]
<div>

  <img />

</div>

// jsx-padded-children: [2, "never"]
<div>
  <img />
</div>
```

Not sure if anyone ever would pick up `always`, that's why I'm also down for `jsx-no-padded-children` or something like that

**Comment by ljharb** [comment]
I think both options are a good idea.

**Comment by joshuakelly** [comment]
@ljharb Okay if i take on this one?

**Comment by ljharb** [comment]
@joshuakelly go for it, that's what the "help wanted" label is for :-)

**Comment by golopot** [comment]
I closed my pr #2211. Feel free to pick this up.

**Comment by mbalc** [comment]
any plans for someone to add this?
